migrations/versions/rev-2023-07-21-09:59:29-e3075e4e1327_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: e3075e4e1327
+Revises: 9881f067213b
+Create Date: 2023-07-21 09:59:29.133119
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'e3075e4e1327'
+down_revision = '9881f067213b'
```

Comment:
Why do you have two migration files in one PR?
Please merge those into one, with the fix for the correct last revision in development mentioned above

Reply:
Hi @norbusan 
because this feature have the dependency with #8958, so another migration file is from that PR.
I will fix it when #8958 is merged.

Reply:
#8958 is merged now. Thanks